doc/glossary.rst

```diff
@@ -1555,9 +1555,12 @@ functions or non-estimator constructors.
         generally be interpreted as ``n_jobs=1``, unless the current
         :class:`joblib.Parallel` backend context specifies otherwise.
 
+        Note that even if ``n_jobs=1``, low-level parallelism (via Numpy and OpenMP)
+        might be used in some configuration.
+
         For more details on the use of ``joblib`` and its interactions with
         scikit-learn, please refer to our :ref:`parallelism notes
-        <parallelism>`.
+        <parallelism>` and :ref:`User Guide <computing>`.
```

Comment:
This `computing` label does not seem to be valid according to the CI.

Reply:
Looking over the user guide, I do not think we need to link to the "Computing" section. The existing link to [Parallelism](https://scikit-learn.org/dev/computing/parallelism.html) is most relevant to `n_jobs`.